tests/plugins/test_polars_extensions.py

```diff
@@ -77,6 +79,24 @@ def test_polars_feather(tmp_path: pathlib.Path) -> None:
     assert metadata["path"] == file_path
 
 
+def test_polars_spreadsheet(df: pl.DataFrame, tmp_path: pathlib.Path) -> None:
+    file = tmp_path / "test.excel"
```

Comment:
this will break.

Reply:
Why do this this will break?

Reply:
🤔  I don't recall. 😅  But I think I might have been referring to how the code looked previously? Otherwise from reading things, you can pass a path and it'll save to it.